app/utils/IncidentP5Helper.scala

```diff
@@ -135,14 +136,23 @@ class IncidentP5Helper(
       call = None
     )
 
-  def endorsementCountryRow: Option[SummaryListRow] =
-    buildRowFromAnswer[String](
-      answer = data.Endorsement.map(_.country),
-      formatAnswer = formatAsText,
-      prefix = "departure.notification.incident.index.endorsementCountry",
-      id = None,
-      call = None
-    )
+  def endorsementCountryRow: Future[Option[SummaryListRow]] =
+    data.Endorsement
```

Comment:
We could maybe consider having a separate helper for the endorsement stuff. With it being an optional data group it would simplify the code a lot to only have to map over that option once